utils/node_local_test.py

```diff
@@ -984,24 +983,24 @@ def run_il_test(server, conf):
 
     pools = get_pool_list()
 
-    # TODO: This doesn't work with two pools, partly related to
-    # DAOS-5109 but there may be other issues.
+    # TODO: This doesn't work with two pools, there appears to be a bug
```

Comment:
(style)  'doesn'' may be misspelled - perhaps 'doesn't'?
(lint) This doesn't work with two pools, there appears to be a bug